nucypher/acumen/nicknames.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, symbol: str, color_name: str, color_hex: str):
         self.color_hex = color_hex
         self._text = color_name + " " + _SYMBOLS[symbol]
 
-    def payload(self):
+    def to_json(self):
```

Comment:
I admit, I do not like this entirely. But there's already `Card.to_json()` that does the same, and it's not just a dict - it's a dict that can be serialized into JSON, which is a small subset of all dicts. I'm open to suggestions. `to_json_serializable`? `json_payload`?

Reply:
Perhaps something like `to_serializable_dict()`; more general than just JSON serialization, or `serializable_payload()`...

Reply:
Changes to the Card API seem out if scope in this PR, especially considering there are open issues that can be solved along with these in a separate changeset.

Reply:
Yep, my recommendation was for this code change, and not for the `Card.to_json` - it can be renamed in a follow-up. I think Bogdan only brought it up as an analogous example somewhere else in the codebase.

Reply:
> I think Bogdan only brought it up as an analogous example somewhere else in the codebase.

Rather, I wanted to reduce the amount of differently-named serialization methods, and looked for an existing one that did what I needed.

I would personally prefer to either keep `to_json()`, and possibly rename it everywhere in another PR, or rename all currently existing occurrences of `to_json()` (including the `Card` one). Just to keep the codebase consistent at each point. 

As for the name, I don't particularly like `to_serializable_dict()`, because it's not necessarily a dict - can be anything serializable to JSON (a list, a string etc).

Reply:
> because it's not necessarily a dict 

I suspect that this was the reason for the original use of `payload()`. You could use what you mentioned before `to_json_serializable()`, it is just that if it is JSON serializable, isn't it just python serializable anyway eg. with `pickle`? Or am I mistaken? In which case, it might as well just be `to_serializable()`.

In any case, I'm good with making all the changes as part of a single and separate PR.

Reply:
> it is just that if it is JSON serializable, isn't it just python serializable anyway eg. with pickle? 

It is. Pretty much anything in Python is pickle-serializable (so it's not even worth an explicit specification), but JSON is much more restrictive. It would be nice to have some term like "simple hierarchical data" that would encompass, say, JSON and YAML, but I'm not aware of any such thing.

Reply:
Filed #2540